image/tarexport/save.go

```diff
@@ -191,15 +189,13 @@ func (s *saveSession) save(outStream io.Writer) error {
 	manifestFileName := filepath.Join(tempDir, manifestFileName)
 	f, err := os.OpenFile(manifestFileName, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
 	if err != nil {
-		f.Close()
 		return err
 	}
+	defer f.Close()
```

Comment:
@vbatts You wouldn't happen to know if the way we use `system.Chtimes()` is particularly magical here?